app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :post
```

Comment:
- [ ] Great job working working on this milestone. I think you have a repetition of this association on lines 2 and 4. kindly delete one of them.